tests/test_core.py

```diff
@@ -6020,10 +6020,10 @@ def test_wasm_builtin_simd(self, js_engines):
 
   @wasm_simd
   def test_wasm_intrinsics_simd(self, js_engines):
-    self.emcc_args.extend(['-Wpedantic', '-Werror', '-Wall'])
+    self.emcc_args.extend(['-Wpedantic', '-Werror', '-Wall', '-xc++'])
     self.do_run(open(path_from_root('tests', 'test_wasm_intrinsics_simd.c')).read(), 'Success!',
                 js_engines=js_engines)
-    self.emcc_args.append('-munimplemented-simd128')
+    self.emcc_args.extend(['-munimplemented-simd128', '-xc', '-std=c99'])
```

Comment:
is there a reason to run the first in c++ and the second in c? would it make sense to run the entire function in both c and c++?

Reply:
We could do that, but these are somewhat long-running tests because compiling the test file takes forever. This seemed sufficient to get the coverage we want. In the future if we remove `-munimplemented-simd128`, I would still want to compile the file once in each language, though.